charts/seed-controlplane/charts/kube-apiserver-service/templates/kube-apiserver-service.yaml

```diff
@@ -6,6 +6,13 @@ metadata:
   {{if eq .Values.cloudProvider "aws" -}}
   annotations:
     service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: "3600"
+    service.beta.kubernetes.io/aws-load-balancer-backend-protocol: "ssl"
+    service.beta.kubernetes.io/aws-load-balancer-ssl-ports: "443"
+    service.beta.kubernetes.io/aws-load-balancer-healthcheck-timeout: "5"
+    service.beta.kubernetes.io/aws-load-balancer-healthcheck-interval: "30"
+    service.beta.kubernetes.io/aws-load-balancer-healthcheck-healthy-threshold: "2"
+    service.beta.kubernetes.io/aws-load-balancer-healthcheck-unhealthy-threshold: "3"
```

Comment:
shouldn't we reduce this to 1 or 2 to remove broken VMs faster?

Reply:
Better to keep it 2. 1 can cause false-positives

Reply:
Agreed, thanks